src/nvim/main.c

```diff
@@ -159,6 +159,7 @@ void event_init(void)
   channel_init();
   terminal_init();
   ui_init();
+  autocmd_init();
```

Comment:
should be done as early as possible. looks like it depends on multique and the event-loop, so just after that. i guess line 156.